src/SensioLabs/Connect/OAuthConsumer.php

```diff
@@ -84,36 +84,36 @@ public function getAuthorizationUri($callbackUri)
     public function requestAccessToken($callbackUri, $authorizationCode)
     {
         $params = array(
-            'client_id'     => $this->appId,
+            'client_id' => $this->appId,
             'client_secret' => $this->appSecret,
-            'code'          => $authorizationCode,
-            'grant_type'    => 'authorization_code',
-            'redirect_uri'  => $callbackUri,
+            'code' => $authorizationCode,
+            'grant_type' => 'authorization_code',
+            'redirect_uri' => $callbackUri,
             'response_type' => 'code',
-            'scope'         => $this->scope,
-            'strict'        => $this->strictChecks,
+            'scope' => $this->scope,
+            'strict' => $this->strictChecks,
         );
 
         $url = sprintf('%s%s', $this->endpoint, $this->paths['access_token']);
 
         $this->logger->info(sprintf("Requesting AccessToken to '%s'", $url));
-        $this->logger->debug(sprintf("Sent params: %s", json_encode($params)));
+        $this->logger->debug(sprintf('Sent params: %s', json_encode($params)));
 
         $response = $this->browser->submit($url, $params);
 
-        $this->logger->debug(sprintf("Response of AccessToken: %s", $response));
+        $this->logger->debug(sprintf('Response of AccessToken: %s', $response));
 
         $content = $response->getContent();
         $response = json_decode($content, true);
 
         if (null === $response) {
-            $this->logger->error('Received non-json response.');
+            $this->logger->error('Received non-json response.', array('response' => $content));
```

Comment:
Did you notice I'm not passing the response object but content.
AFAIK, after re-reading buzz source code, the output of `$response->getContent()` is a string.